Makefile

```diff
@@ -46,6 +46,15 @@ LOCAL_GROUP_ID:=$(shell id -g)
 
 EXTRA_DOCKER_ARGS	+= -e GO111MODULE=on
 
+ifdef GOPATH
```

Comment:
I think this if statement is overkill. Golang treats gopath the same whether it's set to an empty string or unset.

```
(⎈ N/A:N/A)❯ export GOPATH=""                                     (turk@tigera.io:tigera-dev)

~/tmp
(⎈ N/A:N/A)❯ go env GOPATH                                        (turk@tigera.io:tigera-dev)
/home/turk/go

~/tmp
(⎈ N/A:N/A)❯ unset GOPATH                                         (turk@tigera.io:tigera-dev)

~/tmp
(⎈ N/A:N/A)❯ go env GOPATH                                        (turk@tigera.io:tigera-dev)
/home/turk/go
```

Our makefile logic shouldn't care either.

Can we just rely on the following `ifneq` statement?

Reply:
I think so